x/auth/types/account.go

```diff
@@ -43,7 +43,7 @@ func NewBaseAccount(address sdk.AccAddress, pubKey cryptotypes.PubKey, accountNu
 }
 
 // ProtoBaseAccount - a prototype function for BaseAccount
-func ProtoBaseAccount() AccountI {
+func ProtoBaseAccount(_ int64) AccountI {
```

Comment:
Without this change, `simapp#NewSimApp` doesn't pass type check. Although this func isn't used in kava, the `simapp` package is imported in a number of tests.